packages/material-ui/src/styles/createPalette.test.js

```diff
@@ -70,6 +80,51 @@ describe('createPalette()', () => {
       red[700],
       'should use red[700] as the default error dark color',
     );
+    assert.strictEqual(
```

Comment:
I think that we can change most of the tests in this file to be minimal. There is a significant amount of unneeded duplication (even before your changes).

Reply:
Ok, I will try to figure out a way to optimize these tests.